flume/conf/cygnus_instance.conf.template

```diff
@@ -38,3 +38,6 @@ LOGFILE_NAME=cygnus.log
 
 # Administration port. Must be unique per instance
 ADMIN_PORT=8081
+
+# Polling interval for the configuration reloading
```

Comment:
To avoid having to check the code, I would suggest including a comment about the units of the value (seconds, ms., etc.) ;)